# 3DEM_resources
History, software, lecture and more

## History

- [History of 3DEM](https://www.ebi.ac.uk/emdb/genealogy).


## Lectures
- Getting Started in Cryo-EM (Grant Jensen). [Youtube](https://www.youtube.com/playlist?list=PL8_xPU5epJdctoHdQjpfHmd_z9WvGxK8-), [Coursera](https://www.coursera.org/learn/cryo-em?action=enroll).

- [Sample preparation](https://www.youtube.com/playlist?list=PL8_xPU5epJdfd5fM2CjQItR-iRlIEIJk8).
    
- [MRC-LMB Electron Cryo-microscopy Course 2017](https://www.youtube.com/watch?v=aHhmnxD6RCI&list=PLQbPquAyEw4etKtxyqcvZz4uELPeLDLeF).
    
- [SBGrid lectures](https://www.youtube.com/user/SBGridTV).

## softwares

- [A list of the 3DEM softwares on EMDB](https://emdataresource.org/emsoftware.html).
- [A list of the 3DEM softwares on Wiki](https://en.wikibooks.org/wiki/Software_Tools_For_Molecular_Microscopy).
- Image processing
    - [SerialEM](https://bio3d.colorado.edu/SerialEM/). [SerialEM by UMASS](https://readthedocs.org/projects/sphinx-emdocs/downloads/pdf/latest/).
    - [MotionCor2](https://hpc.nih.gov/apps/RELION/MotionCor2-UserManual-05-03-2018.pdf).
    - [CrYOLO](https://cryolo.readthedocs.io/en/stable/).
    - [RELION](https://www3.mrc-lmb.cam.ac.uk/relion/index.php/Main_Page).
    - [CrySPARC](https://cryosparc.com).
    - [DeepEMhancer](https://github.com/rsanchezgarc/deepEMhancer).
    - [CryoDRGN](http://cb.csail.mit.edu/cb/cryodrgn/).
    
- Modeling and visualization
    - [Coot](https://www2.mrc-lmb.cam.ac.uk/personal/pemsley/coot/). 
    - [UCSF Chimera](https://www.cgl.ucsf.edu/chimera/).
    - [UCSF ChimeraX](https://www.cgl.ucsf.edu/chimerax/).
    - [Phenix](https://phenix-online.org).
    - [DeepTracer](https://deeptracer.uw.edu/home).

- Structural prediction
    - [PSIPRED](http://bioinf.cs.ucl.ac.uk/psipred/) - secondary structure prediction. 
    - [SWISS-MODEL](https://swissmodel.expasy.org).
    - [Robetta](https://robetta.bakerlab.org).
    - [AlphaFold](https://alphafold.ebi.ac.uk).

- Tomography
    - [IMOD](https://bio3d.colorado.edu/imod/).
    - [PEET](https://bio3d.colorado.edu/PEET/).
    - [Dynamo](https://wiki.dynamo.biozentrum.unibas.ch/w/index.php/Main_Page).
    - [Wrap/M](http://www.warpem.com/warp/).

- Bioinformatics
    - [A collection on SIB](https://www.expasy.org).
    - [Clustal Omega](https://www.ebi.ac.uk/Tools/msa/clustalo/) - sequence alignment. 
    - [ESPript](https://espript.ibcp.fr/ESPript/ESPript/index.php) - coloring sequence alignment. 
    - [pfam](http://pfam.xfam.org) - domain prediction. 
    - [SMART](http://smart.embl.de) - domain prediction. 
    - [ConSurf](https://consurf.tau.ac.il) - domain prediction. 
    - [CCCP](https://grigoryanlab.org/cccp/) - coiled coil parameters and generator
    - [HOLE](http://www.holeprogram.org) - hole parameters of a channel. 
    - [GLYCAM](http://glycam.org) - glycan prediction. 
 
- Other useful database/tools
    - [HUGO](https://www.genenames.org) - Human gene nomenclature. 
    - [OMIM](https://www.omim.org) - Human Genes and Genetic Disorders. 
    - [The Human Protein Atlas](https://www.proteinatlas.org).
    - [NeXtProt](https://www.nextprot.org) - human protein function annotation. 
    - [TCGA](https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga) - The Cancer Genome Atlas.
    - [ClinVar](https://www.ncbi.nlm.nih.gov/clinvar/) - Information about genomic variation and its relationship to human health.
    - [ConVarT](https://convart.org) - orthologous variants (OrthoVars) between species. 
    - [OrthoMCL](https://orthomcl.org/orthomcl/app/) - grouping orthologous protein sequences.
    - [EVcouplings](https://evcouplings.org) - evolutionary sequence covariation. 
    - [DepMap](https://depmap.org/portal/) - the cancer dependency map. 
    - [Phytozome](https://phytozome.jgi.doe.gov/pz/portal.html) - JGI-sequenced plant genomes at RNA level.
    - [ChlamyFPV5](http://chlamyfp.org) - Annotation of Chlamydomonas proteins.
    - [Human Intermediate Filament Database](http://www.interfil.org/proteins.php).
